clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py

```diff
@@ -72,12 +72,12 @@ def api_call_latency(title, verb, scope, threshold):
     ),
     d.simple_graph(
         "Pod creations",
-        'sum(irate(apiserver_request_total{verb="POST", resource="pods", subresource=""}[1m])) by (client)',
```

Comment:
Let's not get rid of the sum aggregation. We want to see an overall pod creation rate in the cluster, without sum you'll get a separate graph for every unique tuple of prometheus labels, e.g. we can have multiple api-servers with different labels, etc.

Reply:
Fixed, now only `by (client)` is dropped.